boxsdk/session/box_session.py

```diff
@@ -336,68 +312,57 @@ def _prepare_and_send_request(
             `int`
         """
         files = kwargs.get('files')
-        file_stream_positions = None
+        kwargs['file_stream_positions'] = None
```

Comment:
I know this is existing code... a reference to `files` and `file_stream_positions` seems out of the blue here in 100% generic "make a request" code. Might there be a comment explaining why this code is here?